RELEASE_NOTES

```diff
@@ -3,6 +3,64 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
+v2.7.0 (March 18th 2016)
+------------------------
+
+feature + bugfix release
+- stabilize Cray support
+    - enable 'dynamic' toolchain option by default for Cray* toolchains (#1581)
+    - remove FFTW from the Cray Toolchain Definition (#1585)
+    - add external modules metadata for Cray systems (#1638)
+    - fix independency of Cray toolchains w.r.t. toolchain build environment (#1641, #1647)
```

Comment:
no, it's actually *fix*, the independency between Cray toolchains was broken, in the sense that using stuff was leaking from one Cray toolchain into the other, if you were using two different toolchains in a single EB session